src/com/lftechnology/java/training/sanish/student/CalculateGPA.java

```diff
@@ -0,0 +1,96 @@
+package com.lftechnology.java.training.sanish.student;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Calculate GPA of students
+ * 
+ * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+ */
+public class CalculateGPA {
+	private final static Logger LOGGER = Logger.getLogger(CalculateGPA.class
+			.getName());
+	private int studentNumber = 0;
+	private ArrayList<Student> students = new ArrayList<Student>();
+
+	public static void main(String args[]) {
+		CalculateGPA calculateGPA = new CalculateGPA();
+		try (Scanner scanner = new Scanner(System.in)) {
+			LOGGER.log(Level.INFO, "Enter number of students : ");
+			calculateGPA.setStudentNumber(UserInput.getNumber(scanner,
+					Constants.MIN_STUDENT_NUM, Constants.MAX_STUDENT_NUM));
+			calculateGPA.setStudentInformation(scanner);
```

Comment:
is there a reason of passing scanner object as a parameter? Scanner could have been defined static at top and could have used in whole class..